firefox-ios/firefox-ios-tests/Tests/Smoketest1.xctestplan

```diff
@@ -149,6 +149,7 @@
         "HomePageSettingsUITests\/testSetFirefoxHomeAsHome()",
         "HomePageSettingsUITests\/testTopSitesCustomNumberOfRows()",
         "HomePageSettingsUITests\/testTyping()",
+        "InactiveTabsTest",
```

Comment:
This test case is marked as a smoke test, it should be part of a smoke plan. Currently if i am not mistaken is in full functional right?

Reply:
You should remove the test from the full functional plan

Reply:
I agree with that also, but lets make sure we update the test case also.

Reply:
yes, it should be updated in TR too

Reply:
Let me remove the test from smoke for now.